tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -161,12 +161,53 @@ def on_heartbeat(self, cluster_data):
             if data:
                 self.on_sync_object(data)
 
+        # Sync the utilization details for the cluster
+        self._sync_utilization()
+
         # Get and update rbds for pools
         self._sync_rbds()
 
         # Get and update ec profiles for the cluster
         self._sync_ec_profiles()
 
+    def _sync_utilization(self):
+        util_data = self._get_utilization_data()
+        NS.ceph.objects.Utilization(
+            total=util_data['cluster']['total'],
+            used=util_data['cluster']['used'],
+            available=util_data['cluster']['available'],
+            pcnt_used=util_data['cluster']['pcnt_used']
+        ).save(update=False)
+
+        # Loop through the pools and update the utilization details
+        pools = NS._int.client.read(
+            "clusters/%s/Pools" % NS.tendrl_context.integration_id,
+        )
+        for entry in pools.leaves:
+            fetched_pool = NS.ceph.objects.Pool(
+                pool_id=entry.key.split("Pools/")[-1]
+            ).load()
+            pool_used = 0
+            pcnt = 0
+            for pool in util_data['pools']:
+                if pool['name'] == fetched_pool.pool_name:
+                    pool_used = pool['used']
+                    pcnt = pool['pcnt_used']
+            NS.ceph.objects.Pool(
```

Comment:
No need to create new Pool object. Simply make changes to attributes of "fetched_pool" and save() it